contrib/win32/win32compat/termio.c

```diff
@@ -247,6 +245,19 @@ syncio_initiate_write(struct w32_io* pio, DWORD num_bytes)
 	return 0;
```

Comment:
remove the log in the if (0 == QueueUserAPC(ReadAPCProc...) { ... }

Reply:
ok